### DISCUSSION TITLE
Imagen en página

### DISCUSSION THREAD

**Discussion by GerardoSerrato** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#366] [Q&A] [answered]
¿Qué tal están todos?
Tengo un problema en mi página. La imagen que está en el costado me aparece cuando hago pruebas desde VSC, pero al momento de cargar mis archivos a github y montar el sitio, dejó de aparecer, lo que me parece extraño ya que otras imágenes que uso sí aparecen.
Les agradecería mucho si pudieran ayudarme a resolverlo.
[https://gerardoserrato.github.io/index.html](url)
La verdad me está encantando esta clase de prácticas y cómo nos ayuda a seguir formándonos.
![SS](https://user-images.githubusercontent.com/99059849/158034551-5e5c9a04-8028-4a37-b508-62a4d6c23326.jpg)
P.S Pueden darse una vuelta al sitio y dejarme otras sugerencias si es necesario.

**Comment by GerardoSerrato** [comment]
El link está mal, es este
[Página](https://gerardoserrato.github.io/)

**Comment by hackit92** [comment]
Que tal, solo me despliega el banner y no toda la página, es eso correcto?

**Reply by GerardoSerrato** [reply]
Sí! Por ahora solo llevo eso!

**Comment by doguedogue** [comment]
Genial

![Screenshot_2022-03-12-15-42-28-423_com.android.chrome.jpg](https://user-images.githubusercontent.com/23409026/158036016-87082fde-a67a-42c8-9dc5-7dc9a55ae0e3.jpg)

![Screenshot_2022-03-12-15-42-35-376_com.android.chrome.jpg](https://user-images.githubusercontent.com/23409026/158036021-29faa40a-4586-4da2-8cdc-c8885fd1df96.jpg)

**Reply by GerardoSerrato** [reply]
No me había dado cuenta que ya aparece! Gracias!

**Comment by AngelCruzO** [comment, marked as answer]
Github tarda un poco en mostrar las cosas que entran la primera vez o cambios, es el proceso de despliegue, solo ten paciencia y despues aparecen

**Reply by GerardoSerrato** [reply]
Muchas gracias! Nunca había montado un sitio en GitHub ni en ningún otro lado!  Entonces es normal!
